www/js/dashboard.js

```diff
@@ -200,11 +202,11 @@ function changeRobotStrafePos(visionX) {
     if (visionX >= -1.0 && visionX <= 1.0) {
         var robot = document.getElementById("robotSVG");
 
-        var xpos = visionX
-        xpos += 1;
-        xpos = xpos * 50;
-        xpos = xpos + "%";
-        console.log(xpos)
+        var xpos = visionX + 1; // -1 -to 1 now 0 to 2
+            xpos = xpos * 50; // percentage 
+            xpos = xpos * 0.8; // width of picture  
+            xpos = xpos.toString();
+            xpos = xpos.concat("%")
```

Comment:
Don't use `string.concat()` like this, just use the `+` operator.

Also, no need to `.toString()` it either — adding anything with a string always means string concatenation in JS.

Also, why are these lines indented an extra level?